src/main/java/liquibase/addMeasureModelColumn.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet author="mat_dev_user" id="1" context="prod">
+        <sql>
+            ALTER TABLE MEASURE ADD COLUMN MEASURE_MODEL VARCHAR(10) NULL;
```

Comment:
Should this be required? Does it need a default value, for all the legacy data?

Reply:
yeah, I'm thinking this should be default to QDM. But @nicole-hunter just mentioned all existing measures may not be QDM type. so, I'm bit confused

Reply:
Only like 1500 measures have the QDM version filled out which makes me think the older ones aren't qdm.